README.md

```diff
@@ -158,6 +158,29 @@ dnf install golang-bin git
 You should now have everything needed to [try out](#basic-usage) the CLI or hack on the code in ~~vim~~ your favorite
 editor.
 
+### Implementing new package managers
+
+One of Cachi2's core goals is to be a kind of package manager wrapper (around
+e.g. gomod, pip, npm...)
+
+Given the multitude of languages and therefore package managers, contributions
+are **definitely** encouraged.
+
+**Testing**: To ease development, we've implemented a flag
+(`--dev-package-managers`), which will **enable** your in-development package
+manager for test, as long as it is added to the [cachi2/core/resolver.py](cachi2/core/resolver.py)
+`_dev_package_managers` dict; otherwise, it will not be available for testing.
+Invoking it looks like `cachi2 fetch-deps --dev-package-managers yarn`
+
+More explicitly
+
+* `--dev-package-managers` is a flag for `fetch-deps`
+* the language we want to fetch for (in this case, `yarn`) is an argument to `fetch-deps`
```

Comment:
Should we explicitly mention **yarn** here? If the yarn epic is done, we probably need to change this so it is not confusing if yarn is still among dev package managers.